chapters/equations.tex

```diff
@@ -762,6 +762,15 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then!. In this case, the algorithmic statements within the when-statement remain active during the whole
```

Comment:
I suppose there should be an `elsewhen` clarification here as well, preserving the symmetry with the when-equations?